presto-docs/src/main/sphinx/release/release-0.143.rst

```diff
@@ -15,3 +15,6 @@ Hive Changes
 ------------
 
 * Fix native memory leak when reading or writing gzip compressed data.
+* Fix data corruption in :doc:`/sql/create-table-as` when
+  ``hive.respect-table-format`` config is set to false and user-specified
+  storage format disagrees with default.
```

Comment:
I would say "does not match default" or "differs from default" instead of "disagrees"